File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/cafe/CafeEvent.kt
package com.hhp227.concafe.presentation.cafe

sealed interface CafeEvent {
    data object NavigateBack : CafeEvent

    data class NavigateToCast(val id: String) : CafeEvent

    data class NavigateToCafeEvent(val cafeId: String, val eventId: String) : CafeEvent

    data class NavigateToReviewEdit(val cafeId: String, val reviewId: String? = null) : CafeEvent

    data class NavigateToPicture(val imageUrl: String) : CafeEvent

    data object NavigateToSignIn : CafeEvent

    data object ShowReviewDeleteFailedMessage : CafeEvent

    data object ShowReviewReportedMessage : CafeEvent
}

File: iosApp/iosApp/Presentation/Cafe/CafeEvent.swift
//
//  CafeEvent.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/06.
//

import Foundation

enum CafeEvent {
    case navigateBack
    case navigateToCast(id: String)
    case navigateToReviewEdit(cafeId: String, reviewId: String?)
    case navigateToSignIn
    case showReviewDeleteFailedMessage
    case showReviewReportedMessage
}
